CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+cmake_minimum_required(VERSION 3.25)
+project(swift-firebase
+  LANGUAGES Swift)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
+
+# include_directories($<$<COMPILE_LANGUAGE:Swift>:${CMAKE_Swift_MODULE_DIRECTORY}>)
+
+include(FetchContent)
+
+add_library(firebase INTERFACE)
+target_compile_options(firebase INTERFACE
+  "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-Xcc -DINTERNAL_EXPERIMENTAL>")
+target_include_directories(firebase INTERFACE
+  Sources/firebase/include
+  third_party/firebase-development/usr/include)
+target_link_directories(firebase INTERFACE
+  third_party/firebase-development/usr/libs/windows
+  third_party/firebase-development/usr/libs/windows/deps/app
+  third_party/firebase-development/usr/libs/windows/deps/app/external)
+target_link_libraries(firebase INTERFACE
+  crypto
+  firebase_rest_lib
+  flatbuffers
+  libcurl
+  ssl
+  zlibstatic)
+
+add_library(FirebaseCore
+  Sources/FirebaseCore/FirebaseApp+Swift.swift
+  Sources/FirebaseCore/FirebaseConfiguration.swift
+  Sources/FirebaseCore/FirebaseError.swift
+  Sources/FirebaseCore/FirebaseLogging+Swift.swift
+  Sources/FirebaseCore/FirebaseOptions+Swift.swift)
```

Comment:
Probably a dumb question, is there no glob support? We only have a handful of libraries so it likely isn't a huge problem, but this seems like a pain point.

Reply:
There is glob support as long as you don't intend to ever make any changes.  Globs are applied early, so you would never pick up any changes.  TL;DR: NEVER GLOB.